Notification.Application/Commands/Broker/SubscriptionCommand.cs

```diff
@@ -0,0 +1,75 @@
+﻿using MediatR;
+using Notification.Application.Handler;
+using Notification.Application.Logger;
+using Notification.Contracts.DataTransferObject.Broker;
+using Notification.Data.Entities;
+using Notification.Data.Repositories.Interface;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Notification.Application.Commands.Broker
+{
+    public class SubscriptionCommand : IRequest
+    {
+        public SubscriptionCommand(Subscription subscription)
```

Comment:
I suppose better name it's with some verb.
For instance HandleSubscriptionCommand